packages/contracts-ops/contracts/scripts/DeployAccountant.s.sol

```diff
@@ -41,14 +40,30 @@ abstract contract DeployAccountantLogic is Script, Config {
             getAccountantOwner(_domain)
         );
     }
+
+    function updateAccountant(string memory _domain, string memory _configPath)
+        internal
+    {
```

Comment:
beacon and proxy should never be updated, so remove the ability to do so accidentally

Reply:
umm well in this case they never existed before, so they need to be added

Reply:
then the write should be checked so that successive runs of any script do nothing

Reply:
@odyslam best way to achieve this w/ vm.readjson?